src/Particular.CodeRules/tools/install.ps1

```diff
@@ -0,0 +1,6 @@
+﻿param($installPath, $toolsPath, $package, $project)
```

Comment:
@hmemcpy good catch. Will remove the vsixmanifest.

@SimonCropp As @hmemcpy pointed out, as long as the dll is in the package the install only has to run once in order to change the csproj file. Once that's checked in other users just need the package.